### PR TITLE
Fix horizontal scrollbar appearing whereas it should not

### LINKED ISSUE #2337: Page width is too big whereas shouldn't
You can see it in this image (in the search results page):

image

There is a scrollbar although there is no reason for it. It's because of the topbar (width is 718px).

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #2337.

### CODE REVIEW

File: templates/style/_navbar.scss [hunk 1]
```diff
@@ -127,6 +127,7 @@ div.nav-container {
                 box-shadow: none;
                 background-color: var(--color-background);
                 height: 100%;
+                width: 50%;
```
Comment (GuillaumeGomez) [inline templates/style/_navbar.scss:130]: Interestingly enough, whatever the value it has the same size.

**Comment by GuillaumeGomez** [comment]
CI failure is:

> Pulling s3 (minio/minio:)...
> Head "https://registry-1.docker.io/v2/minio/minio/manifests/latest": EOF
> Error: Process completed with exit code 1.

**Comment by syphar** [comment]
@GuillaumeGomez I wanted to reproduce the issue to test the fix, but I can't see a scrollbar in the screenshot, and also when doing searches myself.

**Comment by GuillaumeGomez** [comment]
Reduce the width of the viewport to around 600px (I don't remember the "high" limit from which it appears).

**Review by syphar — APPROVED** [review]

**Pull request merged**